### PR TITLE
Jk auth

### PR DESCRIPTION
**Pull request by jenkens-dev**

### CODE REVIEW

**Comment by jenkens-dev** [comment]
@alphy11

File: src/components/LoginDialogueBox.tsx [hunk 1]
```diff
@@ -22,8 +64,40 @@ const LoginDialogueBox = (props: DialogueProps) => {
...
+        client.writeData({
+          data: {
+            volunteerID: volunteerData.data.loginVolunteer.volunteer.id,
+            token: volunteerData.data.loginVolunteer.token,
+          },
+        });
+        history.push('/volunteer-dashboard');
+      }
+    } else {
+      console.log('nonprofit');
```
Comment (Alphy11) [inline src/components/LoginDialogueBox.tsx]: probably want to remove.  Where's your eslint rule!?

File: src/components/NonprofitCardCollection.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const GET_ALL_NONPROFITS = gql`
 
 const NonprofitCardCollection = () => {
   const { loading, error, data } = useQuery<getNonprofits>(GET_ALL_NONPROFITS);
+  console.log(data);
```
Comment (Alphy11) [inline src/components/NonprofitCardCollection.tsx]: probably don't want this

File: src/components/LoginDialogueBox.tsx [hunk 3]
```diff
@@ -22,8 +64,40 @@ const LoginDialogueBox = (props: DialogueProps) => {
     setPassword(e.target.value);
   };
 
+  const handleSubmit = async (e: SyntheticEvent) => {
```
Comment (Alphy11) [inline src/components/LoginDialogueBox.tsx:67]: ideally, as a user, I would just have a username, and that would be associated to a nonprofit or a volunteer, so you in your API could just know automatically

**Review by Alphy11 — APPROVED** [review]

File: src/components/LoginDialogueBox.tsx (hunk 1 above)
Comment (jenkens-dev) [reply src/components/LoginDialogueBox.tsx]: I don't think I have eslint set up

**Pull request merged**